### DISCUSSION TITLE
Tracking RSS Subscribers

### DISCUSSION THREAD

**Discussion by mertbulan** [plausible/analytics#1356] [Anything Else]
Hello,

I'm super happy to Plausible for months. I have a blog and I know that some of my readers are reading my blog via RSS. Since the RSS content includes the whole blog post, my subscribers don't need to visit my website. So in that case, I am not able know how many people are reading my blog posts. I am wondering if you have any plans to add a feature to track those subscribers.

**Comment by metmarkosaric** [comment]
thanks @mertbulan! is it possible to track RSS subscribers/views through other analytics tools? i don't remember that feature from GA. there was a feedburner thing back in the day that allowed you to track RSS views but otherwise the only thing GA tracked was if you set up custom event to see how many people click on the RSS button on your site to subscribe to your feed (this you can do with Plausible too). asking this because i'm not sure if it's technically possible to track this

**Reply by ukutaht** [reply]
Yeah I'm not sure about this either. RSS readers do not run Javascript on behalf of the content provider as far as I'm aware so I don't think there's a way to include a script in RSS. I am largely ignorant of the standard though, so let me know if there's a way

**Reply by mertbulan** [reply]
That's a good question that I didn't think about. I searched on the internet and seems like it is not possible to track them reliably. The only way to track them is adding a png bug into the description of the feed.xml but seems like RSS services like Feedly fetches the page once for all the users so it is going to be reliable.

**Reply by rosano** [reply]
You could use the Events API server-side https://plausible.io/docs/events-api

This article describes using raw server logs to manually calculate subscribers, and claims that some centralized readers put subscriber counts right in the user agent string https://darekkay.com/blog/rss-subscriber-count/

**Comment by AramZS** [comment]
I don't know if this is a solved problem yet? I was thinking about this recently too and I realized that systems like GA have mostly fallen down on this for a bunch of reasons. The idea of the API is theoretically possible, but the answer from back in the heyday of RSS (which feels like it is coming back) is a tracking pixel. Not the ones we have now that are a whole mess of JS but an actual image pixel that on-download sends an event back to your analytics. @mertbulan sort of gets at this, but the goal is not just grabbing it via the page, but inserting it into each article, so the user downloads it when they open the article and triggers the tracking ping. It would be great to have a privacy-forward way to do this to encourage people to do more RSS on their sites. I imagine having a basic endpoint that serves an image with a url param or path that indicates the source site might be useful.

**Reply by staus** [reply]
I would be a big fan of this! It should be privacy respecting, but it would just be nice to find a way to indicate when feeds articles are being opened from RSS and a pixel sounds like a fine enough way.
